Burkina_Faso/BFA_transform_load_dlt.py

```diff
@@ -82,6 +82,7 @@ def boost_silver():
         .when(col('adm1_name_tmp') == 'Region Etrangere', 'Other')
         .when(col('adm1_name_tmp') == 'Est', 'Est Region Burkina Faso')
         .when(col('adm1_name_tmp') == 'Centre Sud', 'Centre Sud Region Burkina Faso')
+        .when(col('adm1_name_tmp') == 'Centre', 'Centre Region Burkina Faso')
```

Comment:
@nihaas21 as discussed last week, can you verify if `adm1_name` and its definition code (line 79-86) can be safely remove? If so, please remove. If not, let me know why.

Reply:
This works! I accidentally pushed the commit to the other pull request, please delete it from there so I can push it to the correct one.